reference/inventory.v3.yml

```diff
@@ -590,6 +590,9 @@ components:
               type: integer
               description: ID of product.
               example: 120
+            sku_id: 
+              type: integer
+              description: Read-only reference to Catalog V2 API's SKU ID. `null` if the item is a base variant.
```

Comment:
I got this description from the [Get all Variants](https://developer.bigcommerce.com/docs/rest-catalog/product-variants/variants-batch#get-all-variants) endpoint response body, and it mentions the Catalog V2 API. If the Catalog V2 API is deprecated, what's a common use case for this field?

Reply:
The same, read-only reference to V2 API, which is still in use for a lot of public API integration